docs/sql-programming-guide.md

```diff
@@ -1607,13 +1600,13 @@ a regular multi-line JSON file will most often fail.
 
 {% highlight r %}
 # sc is an existing SparkContext.
-sqlContext <- sparkRSQL.init(sc)
+spark <- sparkRSQL.init(sc)
```

Comment:
Currently, `sparkRSQL.init` call `org.apache.spark.sql.api.r.SQLUtils.createSQLContext` which return `SQLContext` object not `SparkSession` object. So here it seems to update the R api ?